### PR TITLE
Added titles and keywords

### PR DESCRIPTION
**Pull request by artisingh2020**

### CODE REVIEW

File: clientresources.json [hunk 1]
```diff
@@ -1,29 +1,186 @@
...
+    },
+    "newdeaths": {
+      "keyword": "newdeaths",
+      "description": "total new deaths on a day"
+    },
+    "totalcases": {
+      "keyword": "totalcases",
+      "description": "total cases on a day"
+    },
+    "totalcasesper1M": {
```
Comment (mattweidman) [inline clientresources.json]: Words like "totalcasesper1M" and "totaldeathsper1M" aren't keywords. Keywords are things the language recognizes, like newdeaths, newcases, deaths, cases, population, first, last, day, and aggregate operations.

File: clientresources.json [hunk 2]
```diff
@@ -1,29 +1,186 @@
...
-    "MaxNewCasesinLast30Days": "max(newcases(day-30,day))",
-    "MinNewCasesinLast7Days": "min(newcases(day-7,day))",
-    "MinNewCasesinLast30Days": "min(newcases(day-30,day))",
-    "TotalRecoveredCasesSoFar": "sum(newcases(first,last))- sum(cases(first,last))"
+    "NewCasesOnaDay": {
+      "title": "New cases on a particuar day",
+      "expression": "newcases(day)"
+    },
+    "NewDeathsOnaDay": {
+      "title": "New Deaths on a particuar day",
```
Comment (mattweidman) [inline clientresources.json]: [nit] should have consistent capitalization

File: clientresources.json (hunk 1 above)
Comment (artisingh2020) [reply clientresources.json]: Correction done.

File: clientresources.json (hunk 2 above)
Comment (artisingh2020) [reply clientresources.json]: Correction done.

**Comment by artisingh2020** [comment]
abandoning this , created new branch to resolve merge conflicts and merged successfully already.

**Pull request closed without merging**